### ISSUE TITLE
Quick suggestion cannot be disabled for some editors

### ISSUE THREAD

**Issue by webczat** [microsoft/vscode#107995]
Hello,
I have vscode insiders, quick suggestions disabled, also autosuggest on trigger characters is disabled.
However, some editors seem to ignore this. I saw this a few times, not sure where, although last time was when editing docker-compose.yml when having docker extension.
When I was typing, suggestions popped up without my permission.
I am not sure where it happens besides this extension, but it was not the only occurrence.

Environment info:
Version: 1.50.0-insider
Commit: 403a6895f401c397933ec5c4f7d89432a33509b6
Date: 2020-09-18T07:13:12.194Z
Electron: 9.3.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Linux x64 5.8.10-arch1-1

**Comment by jrieken** [comment]
> I saw this a few times, not sure where, although last time was when editing docker-compose.yml when having docker extension.

This is too vague for me to start investigating.

**Comment by webczat** [comment]
Sorry for a delay.
Okay this is what I did to reproduce the issue:
* Disable both above mentioned options including quick suggestions and suggestion on trigger characters.
* Install docker extension
* Open an empty folder and create file "docker-compose.yml"
* Start typing: "version: 3" (in my test there was no suggestion at this point) and press enter
* Now try typing something like build. Suggestions should appear even though they were disabled.
Actually when I deleted the file and created it again and started typing "version" suggestions appeared immediately, so it is possible they will appear at first try.
I believe this issue affects all yaml files, but not json files. docker-compose.yml is just an example where it popped up for me.
